pkg/utils/plugin/plugin.go

```diff
@@ -43,3 +43,13 @@ func (reg *ModelRegister) MustGet(name string) inference.Model {
 	}
 	panic("model is not registered")
 }
+
+func (reg *ModelRegister) ListModelNames() []string {
```

Comment:
Is this called anywhere...

Reply:
I will add the caller later. It may be good to list all in the log when controller starts.